### PR TITLE
Travis load mf6

### PR DESCRIPTION
**Pull request by Hofer-Julian**

### CODE REVIEW

**Comment by Hofer-Julian** [comment]
The test fails because the flopy repo seems to be private

**Comment by Hofer-Julian** [comment]
@mjr-deltares from my side this PR is ready to be reviewed now :)

File: autotest/test000_setup.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+def test_build_modflow6_so():
+    # determine if app should be build
+    for idx, arg in enumerate(sys.argv):
+        if arg.lower() == '--nomf6':
+            txt = 'Command line cancel of MODFLOW 6 build'
+            print(txt)
+            return
+
+    pymake.download_and_unzip(url="https://github.com/MODFLOW-USGS/modflow6/archive/develop.zip",
+                              pth="./temp")
```
Comment (mjr-deltares) [inline autotest/test000_setup.py:70]: Hi @Hofer-Julian, shouldn't we use the MODFLOW 6 develop branch for our tests?

Comment (mjr-deltares) [reply autotest/test000_setup.py:70]: O wait, now I see, we _are_ using the develop...

**Pull request merged**